test/Epam.ExternalTraining.Task1.Tests/Task1.1/AnotherTriangleTaskTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Epam.ExternalTraining.Abstractions.Task1_1;
+using Epam.ExternalTraining.Task1.TheMagnificentTen;
+using FluentAssertions;
+using NUnit.Framework;
+using System.Collections.Generic;
+
+namespace Epam.ExternalTraining.Task1.Tests
+{
+	/// <summary> Task 1.1.3 - Another triangle (isosceles triangle) </summary>
+	public class AnotherTriangleTaskTests : ConsoleTestBase
+	{
+		private IAnotherTriangleTask _anotherTriangleTask;
```

Comment:
IAnotherTriangleTask  _blaBlatask;

        [OneTimeSetUp]
        public void Initialize()
        {
            if (_painter == null)
            {
                _blaBlaTask= Assembly.GetAssembly(
                typeof(IAnotherTriangleTask  ))
                .GetTypes()
                .Where(t => t.GetInterfaces()
                .Contains(typeof(IAnotherTriangleTask  )))
                .Select(t => Activator.CreateInstance(t) as IAnotherTriangleTask  )
                .FirstOrDefault();
            }
        }

It's need for providing for students only interfaces without implementation

Reply:
@Marzipan-Coin I think we can leave a stub class that implements the interface with `NotImplementedException`s. No reflection and no problem solution, and the difference of having an empty class already isn't that big.

Reply:
Wow, Nice idea